vertx-opentracing/src/main/java/io/vertx/tracing/opentracing/OpenTracingContext.java

```diff
@@ -0,0 +1,48 @@
+package io.vertx.tracing.opentracing;
+
+import io.opentracing.Span;
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+
+/**
+ * OpenTracingContext adds helpers for associating and disassociating spans with the current {@link
+ * Context}
+ */
+public class OpenTracingContext {
+
+  static final String ACTIVE_SPAN = "vertx.tracing.opentracing.span";
+
+  /**
+   * Get the active span from the current {@link Context}
+   *
+   * @return a {@link Span} or null
+   */
+  public static Span activeSpan() {
```

Comment:
I don't like the `active`, `activate` names here as it can be confused with ScopeManagers. Consider using `set`/`setSpan`.. `get`/...

Reply:
Ok. I'll change those.